app/src/main/java/org/fossasia/openevent/fragments/SessionFragment.java

```diff
@@ -0,0 +1,115 @@
+package org.fossasia.openevent.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.activities.ScheduleSessionDetailActivity;
+import org.fossasia.openevent.adapters.ScheduleSessionsListAdapter;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.dbutils.DbSingleton;
+import org.fossasia.openevent.utils.ISO8601Date;
+import org.fossasia.openevent.utils.IntentStrings;
+import org.fossasia.openevent.utils.RecyclerItemInteractListener;
+import org.fossasia.openevent.utils.SimpleDividerItemDecoration;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by duncanleo on 1/2/16.
+ */
+public class SessionFragment extends Fragment {
+    //TODO: Set to the actual first day of the event
+    public static final long FIRST_DAY_MILLIS = new GregorianCalendar(2015, 4, 5).getTime().getTime();
+
+    private View emptyState;
+    private RecyclerView sessionsRecyclerView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'sessionsRecyclerView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406802118/issues/source?bid=3233850&fileBranchId=3233911#l40)